doc/tutorials/basic/serve.md

```diff
@@ -0,0 +1,232 @@
+# Build Hello World App
+
+In this tutorial, we will develop and *serve* a *Hello World* Panel:
+
+- Serve the app with `panel serve app.py`, `panel serve app.ipynb`, or `panel serve app.md`.
+- serve with auto reload by adding the flag `--autoreload`.
+- stop your server with `CTRL+C`.
+
+## Serve the App
+
+Select a *tab* below and continue
+
+:::::{tab-set}
+
+::::{tab-item} Python Script
+:sync: script
+
+The simplest Panel `.py` app could look like this:
+
+```python
+import panel as pn
+
+pn.extension()
+
+pn.panel("Hello World").servable()
+```
+
+:::{note}
+The code refers to
+
+- `panel`: The Panel python package. It's a convention to import it as `pn`.
+- `pn.extension()`: Loads javascript dependencies and configures Panel.
+- `pn.panel(...)`: Creates a *displayable* Panel component.
+- `.servable()`: Displays the component in a *server app*.
+:::
+
+Copy the code into a file named `app.py`.
+
+Save the file.
+
+Run the Panel server in your terminal with
+
+```bash
+panel serve app.py --autoreload
+```
+
+When the server has started, the terminal output should look like this
+
+```bash
+$ panel serve app.py --autoreload
+2024-01-17 15:49:11,443 Starting Bokeh server version 3.3.2 (running on Tornado 6.3.3)
+2024-01-17 15:49:11,444 User authentication hooks NOT provided (default user enabled)
+2024-01-17 15:49:11,450 Bokeh app running at: http://localhost:5006/app
+2024-01-17 15:49:11,450 Starting Bokeh server with process id: 47256
+```
+
+:::{note}
+The command `panel serve app.py --autoreload` refers to:
+
+- `panel`: the panel executable.
+- `serve`: the command you want Panel to run
+- `app.py`: the file `app.py` you want to serve
+- `--autoreload`: make the server restart after code changes. Use this for **development only**.
+:::
+
+::::
+
+::::{tab-item} Notebook
```

Comment:
I use Panel all the time also for data exploration. I do data exploration with caching and `--autoreload`. I think executing cells and spending time on getting very long, "dirty" notebooks gives you a very inefficient workflow.

But I respect that other users think differently. And I think that one of Panels selling points is that it can fit multiple different users in a team.

The *Explore the Data* section was originally the next section and showcased how valuable Panel is for data exploration in a Notebook.

Reply:
> Any suggestions for how to make Panels value in a notebook clearer?

Showing that Panel apps can be run entirely within a notebook (no `.servable()` + `panel serve ..`) needed.